packages/hardware-trezor/test/transformers/tx.test.ts

```diff
@@ -0,0 +1,189 @@
+import * as Trezor from '@trezor/connect';
+import { CardanoKeyConst, util } from '@cardano-sdk/key-management';
+import {
+  contextWithKnownAddresses,
+  contextWithoutKnownAddresses,
+  knownAddressKeyPath,
+  knownAddressStakeKeyPath,
+  minValidTxBody,
+  txBody,
+  txIn
+} from '../testData';
+import { txToTrezor } from '../../src/transformers/tx';
+
+describe('tx', () => {
+  describe('txToTrezor', () => {
+    test('can map min valid transaction', async () => {
+      expect(
+        await txToTrezor({
+          ...contextWithoutKnownAddresses,
+          cardanoTxBody: minValidTxBody
+        })
+      ).toEqual({
+        additionalWitnessRequests: [],
+        auxiliaryData: undefined,
+        certificates: [],
+        fee: '10',
+        inputs: [
+          {
+            prev_hash: txIn.txId,
+            prev_index: txIn.index
+          }
+        ],
+        mint: undefined,
+        networkId: 0,
+        outputs: [
+          {
+            address:
+              'addr_test1qz2fxv2umyhttkxyxp8x0dlpdt3k6cwng5pxj3jhsydzer3jcu5d8ps7zex2k2xt3uqxgjqnnj83ws8lhrn648jjxtwq2ytjqp',
+            amount: '10'
+          }
+        ],
+        protocolMagic: 999,
+        ttl: undefined,
+        validityIntervalStart: undefined,
+        withdrawals: []
+      });
+    });
+
+    test('can map transaction', async () => {
```

Comment:
maybe lets rename this to "can map more complex transaction"?

Reply:
@DominikGuzei  I imagined this as "can map transaction" in this phase and then to have "can map transaction with scripts / babbage data" as a complex one once we have plutus data. 
 But sure, I can rename it if you think it is better?

Reply:
@tomislavhoracek ok yeah then it's probably fine 👍

Reply:
@DominikGuzei Thanks D :)